# jaguar arm motor command message from DrRobot Robot.
float32 joint_0
float32 joint_1
float32 joint_2
float32 joint_3
float32 joint_4
float32 joint_5